src/main.zig

```diff
@@ -98,23 +79,25 @@ fn executeSummaryCommand(allocator: std.mem.Allocator, month_str: []const u8, ye
         }
     } else {
         std.debug.print("Unknown month: '{s}'\n", .{month_str});
-        return;
+        return 1;
```

Comment:
To me, it seem a little bit inconsistent that we now have the `fatal` function, but still _log and return 1_ manually in most places. The only difference is that it exits immediately instead of returning 1 from the current function, but it's really the same in the end, right? So would you agree that I can just replace all instances of _log and return 1_ with calls to `fatal`?
My change would be this https://github.com/GigaGrunch/trecker/commit/83a906e9c251c40fe369ab1ffa7a3e64454c2b92

Reply:
Yeah I was trying to pick up a vibe with which one you'd prefer but I ended up doing both...lol! Intend to just call exit myself.